src/config/updateAxis.js

```diff
@@ -0,0 +1 @@
+export default (axis, options) => axis.update(options);
```

Comment:
Isn't this an overkill? :)
updateAxis(axis, options)
vs
axis.update(options)
:)